Team-Kangaroo/space-barter-backend
# [ADD] phone verification

#### What does this PR do?
Allow users to verify their account with codes via SMS.

#### How should this be manually tested?
The endpoint to hit is: `POST /verify?phone=2348159XXXXXX`. The user receives a 6 digit code for verifying their account.

Adding an extra query like this: `/POST /verify?phone= 2348159XXXXXX&method=call` will initiate verification via `phone call` instead of `sms`.

#### Any background context you want to provide?
Enter phone numbers like so: `2348159XXXX (Nigeria), 1409892XXX (North America), 44703281XXX(United Kingdom)`, etc.
